fancy_prompt/fancy_prompt.py

```diff
@@ -185,7 +187,7 @@ def prompt_hook(current_prompt):
     prompt_extra = ' '.join(fn() for fn in prompt_components)
 
     # Interpolate extra components into our base prompt string.
-    return '(udb {}) '.format(prompt_extra)
+    return '(udb {}) '.format(gdb.prompt.substitute_prompt(prompt_extra))
```

Comment:
I didn't know this existed! Where did you find it?

Did you look at the code?
It looks like `\[` just gets translated into `\001` and `\]` into `\002`.

Reply:
It was in the original code. I removed it because I didn't think it was needed.